satellite/metainfo/metainfo.go

```diff
@@ -599,3 +599,38 @@ func (endpoint *Endpoint) ProjectInfo(ctx context.Context, req *pb.ProjectInfoRe
 		ProjectSalt: salt[:],
 	}, nil
 }
+
+// CreateBucket creates a bucket
+func (endpoint *Endpoint) CreateBucket(ctx context.Context, req *pb.BucketCreateRequest) (_ *pb.BucketCreateResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+	// TODO: placeholder to implement pb.MetainfoServer interface.
+	return &pb.BucketCreateResponse{}, err
```

Comment:
Nit: these all should return errors for now. e.g. `status.Error(codes.Unimplemented, "not implemented")`

Reply:
i have another PR going and i can update this